package.json

```diff
@@ -37,9 +40,12 @@
     "shelljs": "~0.3.0"
   },
   "devDependencies": {
+    "chai": "^2.0.0",
+    "jscoverage": "^0.5.9",
```

Comment:
Not an expert here. Based on above stats I would definitely pick Istanbul but I also want to hear more from @GianlucaGuarini why he prefers jscoverage. Just more familiar with it?